Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -66,23 +98,50 @@ public KeyCommand()
             }
         }
 
-        [Command(Description = "Import a raw private key.")]
+        [Command(Description = "Import a raw private key or Web3 Secret Storage.")]
         public void Import(
             [Argument(
                 "PRIVATE-KEY",
-                Description = "A raw private key to import in hexadecimal string."
+                Description = "A raw private key in hexadecimal string, or path to Web3 Secret " +
+                              "Storage to import"
             )]
             string key,
             PassphraseParameters passphrase,
             [Option(
-                Description = "Print the created private key as Web3 Secret Storage format."
+                Description = "Import Web3 Secret Storage key."
             )]
             bool json = false,
             [Option(Description = "Do not add to the key store, but only show the created key.")]
-            bool dryRun = false
+            bool dryRun = false,
+            [Option(Description = "Path to key store")]
+            string? path = null
         )
         {
-            Add(key, passphrase, json, dryRun, key, false);
+            if (path != null)
+            {
+                KeyStore = new Web3KeyStore(Path.GetFullPath(path));
+            }
```

Comment:
The `--path` option and the logic to override `KeyStore` seem used in almost all commands. How about abstracting it with `ICommandParameterSet` like `PassphraseParameters`?

Reply:
In my perspective, abstracting as `ICommandParameterSet` may harm the consistency of behaviour DefaultStore being as a singleton, and bit complicated for simple overriding, but It's also true that codes are duplicated, so I just made the `ChangeKeyStorePath(path)` function and put the logic inside of it. this way we can ignore both scope issue of using interface and duplicated code.